mirror-service/src/main/java/org/acme/nvd/NvdMirrorHandler.java

```diff
@@ -0,0 +1,45 @@
+package org.acme.nvd;
+
+import io.github.jeremylong.nvdlib.nvd.DefCveItem;
+import org.acme.client.NvdClient;
+import org.cyclonedx.model.Bom;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.enterprise.context.ApplicationScoped;
+import javax.inject.Inject;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+
+@ApplicationScoped
+public class NvdMirrorHandler {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(NvdMirrorHandler.class);
+    private NvdClient client;
+    List<Bom> nvdVulnerabilities;
+    @Inject
+    public NvdMirrorHandler(NvdClient client) {
+        this.client = client;
+        nvdVulnerabilities = new ArrayList<>();
+    }
+
+    public List<Bom> performMirror() {
+        long start = System.currentTimeMillis();
+        LOGGER.info("Starting NVD mirroring task");
+        getNvdFeeds();
+        long end = System.currentTimeMillis();
+        LOGGER.info("NVD mirroring complete");
```

Comment:
should we have something like this to just keep it in one line and keep it aligned with other messages         
`LOGGER.info("NVD mirroring complete. Time spent (total): {} ms", (end - start));`